mission-vision.md

```diff
@@ -0,0 +1,30 @@
+# OpenFeature Mission and Vision
+
+## Mission
+
+To improve the software development lifecycle, no matter the size of the project, by standardizing feature flagging for developers.
```

Comment:
The statement `To improve the software development lifecycle`, is too general. 

What about something like

OpenFeature's Mission: **remove feature flag adoption barriers by standardizing feature flagging for developers.**

Reply:
I hear what you're saying but I personally like that it's a bit ambitious. I think standardized feature flagging can lead to improvements across all aspects of the SDLC.